charts/cert-manager-issuers/examples/letsencrypt-clusterissuers.yaml

```diff
@@ -0,0 +1,25 @@
+# Default Let's Encrypt ClusterIssuers
+#
+# Configures the common letsencrypt-staging and letsencrypt-prod ClusterIssuers
+# using a http01 solver that will work well with nginx-ingress. This is what
+# most small single-tenant clusters start with.
+
+_: &email user@example.com
```

Comment:
What is this? 🤔

Reply:
late-stage-yaml.png

A YAML anchor that is used so the string "user@example.com" is only once in the file. It uses `_` as a dummy key since it looks like the Go blank operator.